codespell_lib/data/dictionary.txt

```diff
@@ -23751,6 +23751,7 @@ seige->siege
 seing->seeing
 seinor->senior
 seires->series
+sekected->selected
```

Comment:
Would you mind adding the select and selects versions of this typo too please.

Reply:
Done. I also added the appropriate plurals for the other lines changed in this PR.